doc/whatsnew/fragments/2018.user_action

```diff
@@ -4,17 +4,20 @@ and name length, and users regularly reported this to be surprising.)
 
 If checking for a minimum length is still desired, it can be regained in two ways:
 
-- If you are content with a ``disallowed-name`` message (instead of ``invalid-name``),
-then simply add the option ``bad-names-rgxs="^..?$"``, which will fail 1-2
-character-long names. (Ensure you enable ``disallowed-name``.)
+- If you are content with a ``disallowed-name`` message (instead of
+  ``invalid-name``), then simply add the option ``bad-names-rgxs="^..?$"``,
+  which will fail 1-2 character-long names. (Ensure you enable
+  ``disallowed-name``.)
 
-- If you would prefer an ``invalid-name`` message to be emitted, or would prefer
-finer-grained control over the circumstances in which messages are emitted
-(classes vs. methods, etc.), then avail yourself of the regex options described
-`here <https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#main-checker>`_.
-(In particular, take note of the commented out options in the "example configuration" given at
-the bottom of the section.) The prior regexes can be found in the
-`pull request <https://github.com/pylint-dev/pylint/pull/8813>`_
-that removed the length requirements.
+- If you would prefer an ``invalid-name`` message to be emitted, or would
+  prefer finer-grained control over the circumstances in which messages are
+  emitted (classes vs. methods, etc.), then avail yourself of the regex
+  options described
+  `here <https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#main-checker>`_.
```

Comment:
Unfortunately, this still won't work because `towncrier` will split this line before the URL and then lose the indentation on the next line.

Reply:
We could turn off the `wrap = True` in the config...

Reply:
I've encountered the issue too, very annoying. I've used a less than ideal formatting but I'd be okay with changing the expectation on the fragment directly in tbump, maybe creating title and text so the text can be anything? It seems a lot of project use something really automated (ruff for example) but I did not have any time to go check how it's one. (ruff's benchmark is also very tasty if I had to prioritize). Maybe that formatting issue is not that important of a problem to fix ?

Reply:
I have no preference. I just don't know how to proceed other than removing the link or setting wrap = False or creating a tinyUrl for it lol

Reply:
I did not check in detail but I'm ok with removing constraints directly if possible. ``wrap = False`` will be easier maybe ?